ledger/acctonline.go

```diff
@@ -181,7 +181,7 @@ func (ao *onlineAccounts) initializeFromDisk(l ledgerForTracker, lastBalancesRou
 		return
 	}
 
-	ao.accountsq, err = ao.dbs.MakeOnlineAccountsReader()
+	ao.accountsq, err = ao.dbs.MakeOnlineAccountsOptimizedReader()
```

Comment:
what is a difference between MakeOnlineAccountsOptimizedReader and MakeOnlineAccountsReader ? They both call OnlineAccountsInitDbQueries

Reply:
thats because they are not 2 different functions, it was renamed. 
The TrackerDB interface had the wrong naming scheme.